test/distributed/test_c10d_nccl.py

```diff
@@ -2162,14 +2189,29 @@ def test_ddp_checkpointing_twice(self):
                     use_bucket_view=use_bucket_view,
                     static_graph=False,
                 )
+                model = self._test_ddp_checkpointing(
+                    self.CheckpointTwiceModule(),
+                    process_group=process_group,
+                    use_bucket_view=use_bucket_view,
+                    static_graph=False,
```

Comment:
How does this work with static_graph=False?

Reply:
It doesn't, if you expand the code above this is in a `assertRaisesRegex` block. This particular test is verifying with static_graph=False, and for both unused param settings, the checkpointing twice fails because we expect to mark a variable ready only once.